applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -407,6 +414,26 @@ static void scan_init(void)
 	k_delayed_work_init(&scan_stop_trigger, scan_stop_trigger_fn);
 }
 
+static void enable_llpm(struct bt_conn *conn)
+{
+	if (IS_ENABLED(CONFIG_BT_LL_NRFXLIB)) {
+		struct bt_le_conn_param param = {
+			.interval_min = 0x0D01,
+			.interval_max = 0x0D01,
+			.latency = 99,
+			.timeout = 400
+		};
+
+		int err = bt_conn_le_param_update(conn, &param);
```

Comment:
Please use the vendor specific HCI API for doing the connection parameter update. See 
https://github.com/NordicPlayground/nrfxlib/blob/master/ble_controller/include/ble_controller_hci_vs.h#L42 and https://github.com/NordicPlayground/nrfxlib/blob/master/ble_controller/include/ble_controller_hci_vs.h#L164.

Reply:
Thanks for the information! I will add confirm options for number of connections and change API as you suggest.

Reply:
Discussed offline - I cannot do this since Zephyr provides no API to get the connection handler. I have talked with @joerchan and he will add the API for us. Once it is available we will update the code. For now we need to stay with what is. It does work so we can live with it for week or two.